packages/truffle-workflow-compile/test/index.js

```diff
@@ -25,30 +36,28 @@ describe("Contracts.compileSources", () => {
 });
 
 describe("Contracts.compile", () => {
-  it("when config.all is true, all contracts in contracts_directory are recompiled", () => {
+  it("when config.all is true, all contracts in contracts_directory are recompiled", async () => {
     let contractName;
 
     // initial compile
-    Contracts.compile(config, (err, { contracts }) => {
+    await Contracts.compile(config, (err, { contracts }) => {
```

Comment:
This should either take a callback or use a Promise interface. If you use `await`, then this should be 
```
const { contracts } = await Contracts.compile(config);
```

Reply:
Sweet, thanks for clarifying the interface stuff @eggplantzzz !